README.md

```diff
@@ -53,24 +53,33 @@ Help for `alchemical_analysis.py` (obtained with `alchemical_analysis -h`) is:
   -h, --help            show this help message and exit
   -a SOFTWARE, --software=SOFTWARE
                         Package's name the data files come from: Gromacs,
-                        Sire, or AMBER. Default: Gromacs.
+                        Sire, Desmond, or AMBER. Default: Gromacs.
+  -b BREAKS, --break=BREAKS
```

Comment:
I don't understand the "breaks" option -- neither why it's called "breaks" nor what the doc string means. Provide a bit more detail and a usage example, maybe? Also, for mnemonic value, the doc string should at least use the word "break" or "breaks".

Reply:
(I'm assuming this is some sort of a fraction since the default is 1.0, but it's very unclear.)

Reply:
@davidlmobley initially I am thinking to use break to match -b as they all start with letter b. Yes, it's the fraction and I change the option word to fraction and update the doc string.